salt/etc-hosts/hosts.jinja

```diff
@@ -1,10 +1,13 @@
 {% set internal_fqdn = "api api." + pillar['internal_infra_domain'] -%}
+{% set external_fqdn = "" -%}
+{% if not salt['caasp_filters.is_ip'](pillar['api']['server']['external_fqdn']) %}
```

Comment:
This is not the most efficient way to do it, as we are querying the pillar twice...

Reply:
Pillar data is a in memory dictionary at this point - readability is significantly more important than an extra dictionary lookup IMO.